_faq/installation_and_deployment/run-forseti-without-org-access.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Can I install and run Forseti without Org Access?
+order: 6
+---
+{::options auto_ids="false" /}
+
+Forseti is designed to install and run out-of-the box with complete
+org access, but you can install Forseti if you aren't an org admin. You'll
+then manually give Forseti the permissions to inventory and audit a subset
+of resources by one specific folder or by projects that are directly under
+the org. When you run Forseti without org access, Forseti Explain and
+Forseti IAM scanner will not contain the full results. This is because
+of Cloud IAM policy inheritance and limited access of the service account.
+
+If you aren't an org admin and an org admin isn't available to grant org
+access to Forseti, follow the process below:
+
+   1. Run the Forseti [installer]({% link _docs/latest/setup/install.md %}). The installer will try to assign org-level. You can ignore the org-level role assignment.
+   1. The installer will create all the necessary Forseti resources: Forseti project, VM instances, Cloud SQL database, and most importantly the +project, VM instances, Cloud SQL database, and service accounts.
```

Comment:
What is +project and why is it the most important thing here?

Reply:
That was a mistake, updated to 
"Forseti project, VM instances, CloudSQL database, Google Cloud Storage buckets and service accounts."